lib/libspl/include/assert.h

```diff
@@ -72,7 +72,8 @@ do {									\
 	const TYPE __right = (TYPE)(RIGHT);				\
 	if (!(__left OP __right))					\
 		libspl_assertf(__FILE__, __FUNCTION__, __LINE__,	\
-		    "%s %s %s (0x%llx %s 0x%llx)", #LEFT, #OP, #RIGHT);	\
+		    "%s %s %s (0x%llx %s 0x%llx)", #LEFT, #OP, #RIGHT,	\
+		    __left, #OP, __right);				\
```

Comment:
Good catch!  You might want to cast the args like in the old code:
`(u_longlong_t)__left, #OP, (u_longlong_t)__right);`